Source/WebKit/WebKitSwift/WritingTools/IntelligenceTextEffectViewManager.swift

```diff
@@ -0,0 +1,195 @@
+// Copyright (C) 2025 Apple Inc. All rights reserved.
+//
+// Redistribution and use in source and binary forms, with or without
+// modification, are permitted provided that the following conditions
+// are met:
+// 1. Redistributions of source code must retain the above copyright
+//    notice, this list of conditions and the following disclaimer.
+// 2. Redistributions in binary form must reproduce the above copyright
+//    notice, this list of conditions and the following disclaimer in the
+//    documentation and/or other materials provided with the distribution.
+//
+// THIS SOFTWARE IS PROVIDED BY APPLE INC. AND ITS CONTRIBUTORS ``AS IS''
+// AND ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT LIMITED TO,
+// THE IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS FOR A PARTICULAR
+// PURPOSE ARE DISCLAIMED. IN NO EVENT SHALL APPLE INC. OR ITS CONTRIBUTORS
+// BE LIABLE FOR ANY DIRECT, INDIRECT, INCIDENTAL, SPECIAL, EXEMPLARY, OR
+// CONSEQUENTIAL DAMAGES (INCLUDING, BUT NOT LIMITED TO, PROCUREMENT OF
+// SUBSTITUTE GOODS OR SERVICES; LOSS OF USE, DATA, OR PROFITS; OR BUSINESS
+// INTERRUPTION) HOWEVER CAUSED AND ON ANY THEORY OF LIABILITY, WHETHER IN
+// CONTRACT, STRICT LIABILITY, OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE)
+// ARISING IN ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF
+// THE POSSIBILITY OF SUCH DAMAGE.
+
+#if canImport(WritingTools)
+
+import Foundation
+import WebKit
+import WebKitSwift
+@_spi(Private) import WebKit
+
+@MainActor
+class IntelligenceTextEffectViewManager<Source> where Source: PlatformIntelligenceTextEffectViewSource, Source.Chunk == IntelligenceTextEffectChunk {
+    init(source: Source? = nil, contentView: PlatformView) {
+        self.source = source
+        self.contentView = contentView
+    }
+
+    private weak var source: Source?
+    private let contentView: PlatformView
+
+    private var effectView: PlatformIntelligenceTextEffectView<Source>? = nil
+
+    // Use the corresponding setter functions instead of setting these directly.
```

Comment:
not something you can enforce instead of leaving this comment? they are private, isn't that enough?

Reply:
(everything in this file is basically just copy-pasted from the original file) but yes now that it's own type and they are private I can probably just remove the comment

Reply:
(everything in this file is basically just copy-pasted from the original file) but yes now that it's own type and they are private I can probably just remove the comment